actix-web/custom-tracing-subscriber/Cargo.toml

```diff
@@ -0,0 +1,12 @@
+[package]
+name = "custom-tracing-subscriber"
+version = "0.1.0"
+edition = "2021"
+
+[dependencies]
+actix-web = "4.3.1"
+shuttle-actix-web = "0.25.0"
+shuttle-runtime = { version = "0.25.0", default-features = false }
```

Comment:
Maybe a comment here too about how we disable the default features to disable the shuttle subscriber setup.

Reply:
Done in fba420bcde99415e11dbbb03dacc78b95c72acc1